portfolio/src/main/java/com/google/sps/data/ReviewDaoDatastore.java

```diff
@@ -128,8 +129,10 @@ public void updateReview(Review review) throws EntityNotFoundException {
    * @return Corresponding Review object that is created
    */
   private Review entityToReview(Entity reviewEntity) {
+    Text contentAsText = (Text) reviewEntity.getProperty(FULLTEXT_PROPERTY);
+    String content = contentAsText.getValue();
     return Review.builder()
-        .fullText((String) reviewEntity.getProperty(FULLTEXT_PROPERTY))
+        .fullText(content)
```

Comment:
Why separate this into three lines? I don't so contentAsText or content re-used anywhere, so they don't really add much value existing as their own variables.

Could we just stick with it how it was before? Or combine it all into one line without the intermediate variables?